packages/phone-number-privacy/combiner/src/domain/endpoints/quota/action.ts

```diff
@@ -0,0 +1,34 @@
+import { DomainQuotaStatusRequest, ErrorMessage } from '@celo/phone-number-privacy-common'
+import { CombineAction } from '../../../common/combine'
+import { IO } from '../../../common/io'
+import { Session } from '../../../common/session'
+import { OdisConfig } from '../../../config'
+import { DomainThresholdStateService } from '../../services/thresholdState'
+
+export class DomainQuotaAction extends CombineAction<DomainQuotaStatusRequest> {
+  constructor(
+    readonly config: OdisConfig,
+    readonly thresholdStateService: DomainThresholdStateService<DomainQuotaStatusRequest>,
+    readonly io: IO<DomainQuotaStatusRequest>
+  ) {
+    super(config, io)
+  }
+
+  async combine(session: Session<DomainQuotaStatusRequest>): Promise<void> {
+    if (session.responses.length >= this.config.keys.threshold) {
+      try {
+        const domainQuotaStatus = this.thresholdStateService.findThresholdDomainState(session)
+        this.io.sendSuccess(200, session.response, domainQuotaStatus)
+        return
+      } catch (error) {
+        session.logger.error({ error }, 'Error combining signer quota status responses')
```

Comment:
same with this function (re: logging); either remove the {} or rename the param to be `err`